languages/tree-sitter-stack-graphs-java/Cargo.toml

```diff
@@ -36,5 +36,6 @@ harness = false # need to provide own main function to handle running tests
 [dependencies]
 anyhow = "1.0"
 clap = { version = "4", features = ["derive"] }
+stack-graphs = { version = "0.12", path = "../../stack-graphs" }
```

Comment:
Is this spurious?  I don't see a change in the crate's Rust code that needs to refer to `stack-graphs` directly

Reply:
Good catch!